stable/concourse/templates/web-deployment.yaml

```diff
@@ -51,6 +51,20 @@ spec:
                   name: {{ template "concourse.fullname" . }}
                   key: postgresql-uri
             {{ end }}
+            {{ if .Values.encryptionKey }}
```

Comment:
No. But the image tag can be overridden, and if using version 2.6.0 and no encryption key specified nothing changes.
Perhaps the default tag version should be updated (2.6.0 is outdated by now), but this is another change.